packages/venia-concept/src/ServiceWorker/__tests__/registerRoutes.spec.js

```diff
@@ -63,6 +67,76 @@ test('There should be a route for robots.txt, favicon.ico and manifest.json with
     registerRoute.mockClear();
 });
 
+test('There should be a route for resized images without cache handler', async () => {
+    const mockUrl = 'mockUrl';
+    const mockRequest = jest.fn();
+    const mockEvent = {
+        waitUntil: jest.fn()
+    };
+    const mockData = {
+        url: mockUrl,
+        request: mockRequest,
+        event: mockEvent
+    };
+
+    const mockFindSameOrLargerImage = jest.fn((url, request) => {
+        return Promise.resolve({ url, request });
+    });
+
+    findSameOrLargerImage.mockImplementation(mockFindSameOrLargerImage);
+
+    registerRoutes();
+
+    const [, handlerCall] = registerRoute.mock.calls[1];
+    const registerResult = await handlerCall(mockData);
+
+    expect(mockFindSameOrLargerImage).toHaveBeenCalled();
+    expect(registerResult).toStrictEqual({
+        url: mockUrl,
+        request: mockRequest
+    });
+
+    registerRoute.mockClear();
+});
+
+test('There should be a route for resized images with cache handler', async () => {
+    const mockUrl = 'mockUrl';
+    const mockRequest = jest.fn();
+    const mockEvent = {
+        waitUntil: jest.fn()
+    };
+    const mockData = {
+        url: mockUrl,
+        request: mockRequest,
+        event: mockEvent
+    };
+
+    const mockFindSameOrLargerImage = jest.fn(() => {
+        return Promise.resolve(null);
+    });
+
+    findSameOrLargerImage.mockImplementation(mockFindSameOrLargerImage);
+
+    const mockCacheHandler = jest.fn(data => data);
+
+    createImageCacheHandler.mockImplementation(() => ({
```

Comment:
Is there a missing `expect` for checking if the cache handler was called?

Reply:
@JustinConabreeAbsolunet You're right. I've added the missing line that checked if it was called. Thanks for the heads up!